german/Alkohol/alkohol.md

```diff
@@ -4,3 +4,12 @@ können sich jedoch seitdem an nichts mehr erinnern. Schließlich hatten Sie ach
 Vorab tranken sie bereits 5 Bier.
 
 Was ist passiert?
+
+Ihr Schädel brummt - wenn das Wasser um Sie herum nicht ärgerlicher Weise Salzwasser wäre, würden sie sich einen großen Schluck davon gönnen.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.